test/verify/check-journal

```diff
@@ -403,5 +403,29 @@ s.send("PRIORITY=3\\nFOO=bar\\n")'
         b.wait_visible(sel)
         self.assertIn("/reports/bthash/123deadbeef", b.attr(sel, 'href'))
 
+        # "Unreport" the problem to test reportin unknown problem
+        m.execute('find /var/spool/abrt -name "reported_to" | xargs rm')
+
+        b.reload()
+        b.enter_page("/system/logs")
+
+        sel = "#journal-entry-fields"
+        b.wait_present(sel)
+
+        self.allow_journal_messages('This problem has already been reported.')
+        self.allow_journal_messages('http://localhost:12345/reports/42/')
+        self.allow_journal_messages('https://bugzilla.example.com/show_bug.cgi\?id=123456')
```

Comment:
I think this needs a `.*` at the beginning, to account for the `cockpit-bridge[1346]:` prefix. The test case doesn't [get that far](https://fedorapeople.org/groups/cockpit/logs/pull-7244-20170713-123030-ae1b2145-verify-fedora-26/log.html#71) as it's already skipping the allowed journal messages for issue #6108, but it would be more robust to add it.